lib/infra_conversion_throttler.rb

```diff
@@ -1,24 +1,44 @@
 class InfraConversionThrottler
+  include Vmdb::Logging
+
   def self.start_conversions
+    _log.debug("InfraConversionThrottler.start_conversions")
     pending_conversion_jobs.each do |ems, jobs|
+      _log.debug("- EMS: #{ems.name}")
+      _log.debug("-- Number of pending jobs: #{jobs.size}")
       running = ems.conversion_hosts.inject(0) { |sum, ch| sum + ch.active_tasks.count }
-      $log&.debug("There are currently #{running} conversion hosts running.")
+      _log.debug("-- Currently running jobs in EMS: #{running}")
       slots = (ems.miq_custom_get('Max Transformation Runners') || Settings.transformation.limits.max_concurrent_tasks_per_ems).to_i - running
-      $log&.debug("The maximum number of concurrent tasks for the EMS is: #{slots}.")
+      _log.debug("-- Available slots in EMS: #{slots}")
       jobs.each do |job|
+        vm_name = job.migration_task.source.name
+        _log.debug("- Looking for a conversion host for task for #{vm_name}")
+
+        if slots <= 0
```

Comment:
Could this be outside the `jobs.each` loop?  It looks like there is no context in this loop that impacts the slots check and it breaks out anyway

Reply:
Yes, it could. We talked about it with Keenan, but I wanted to limit the change to new log messages.
Anyway, it's a good point, so I updated the code.